packages/terra-divider/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Divider migrated from horizontal rule to div
```

Comment:
We should be fine dropping this "Changed" section since we rendering the HR now in the default case.

Reply:
Changed: [81179c5](https://github.com/cerner/terra-core/pull/2296/commits/81179c5398cb4eeb8c820f4f76ec2d946aaea09b)